lib/rubocop/result_cache.rb

```diff
@@ -78,7 +79,7 @@ def self.cleanup(config_store, verbose, cache_root = nil)
 
     def self.cache_root(config_store)
       root = config_store.for('.')['AllCops']['CacheRootDirectory']
-      root = Dir.tmpdir if root == '/tmp'
+      root = File.join(Dir.tmpdir, Etc.getlogin) if root == '/tmp'
```

Comment:
see reply back in thread:

/tmp/rubocop_cache/brenda won't work in some scenarios. The first user will make and own /tmp/rubocop_cache, and later users may be unable to write to create a sub dir.